service/src/test/java/bio/terra/workspace/service/resource/controlled/ControlledResourceServiceTest.java

```diff
@@ -2042,7 +2060,12 @@ private ControlledBigQueryDatasetResource makeBigQueryDatasetWithLifetime(
                 resource, null, user.getAuthenticatedRequest(), creationParameters)
             .castByEnum(WsmResourceType.CONTROLLED_GCP_BIG_QUERY_DATASET);
 
-    assertEquals(resource, createdDataset);
+    assertEquals(resource.getResourceId(), createdDataset.getResourceId());
+    assertEquals(datasetId, createdDataset.getDatasetName());
+    assertEquals(projectId, createdDataset.getProjectId());
+    assertEquals(defaultTableLifetime, createdDataset.getDefaultTableLifetime());
+    assertEquals(defaultPartitionLifetime, createdDataset.getDefaultPartitionLifetime());
+
```

Comment:
No longer using deep equals comparison. We should only care if the resource ID, and the attributes match up (most importantly, the lifetime)